services/repository/src/main/java/org/sagebionetworks/repo/web/controller/MembershipInvitationController.java

```diff
@@ -165,4 +177,41 @@ public void deleteInvitation(
 			) {
 		return serviceProvider.getMembershipInvitationService().getOpenInvitationCount(userId);
 	}
+
+	/**
+	 * Verify whether the inviteeEmail of the indicated MembershipInvitation is associated with the authenticated user.
+	 * If it is, the response body will contain an InviteeVerificationSignedToken.
+	 * If it is not, the response body will be null.
+	 * @param membershipInvitationId
```

Comment:
Please consider adding a link to the design document.

Reply:
An alternative pattern:  let this service be an HTTP GET which returns the InviteeVerificationSignedToken (status=200 OK) if the authenticated user is associated with the embedded email or 403 Unathorized if the user is anonymous or if the embedded email address is NOT associated with the authenticated user.